microstream/src/test/groovy/io/micronaut/microstream/health/MicrostreamHealthIndicatorDetailSpec.groovy

```diff
@@ -0,0 +1,38 @@
+package io.micronaut.microstream.health
+
+import groovy.json.JsonSlurper
+import io.micronaut.context.ApplicationContext
+import io.micronaut.http.client.HttpClient
+import io.micronaut.runtime.server.EmbeddedServer
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.TempDir
+
+class MicrostreamHealthIndicatorDetailSpec extends Specification {
+
+    @TempDir
+    @Shared
+    File tempDir
+
+    def "details are shown for the health endpoint if requested"() {
+        given:
+        EmbeddedServer server = ApplicationContext.run(EmbeddedServer, [
+                "microstream.storage.blue.storage-directory": tempDir.absolutePath,
+                "endpoints.health.details-visible": "ANONYMOUS",
+        ])
+        def client = server.applicationContext.createBean(HttpClient, server.URL).toBlocking()
```

Comment:
`BlockingHttpClient` instead of `def`

Reply:
resolved via https://github.com/micronaut-projects/micronaut-microstream/pull/22/commits/5c76dc9c208a8c8b44674eaa5404d87c29b29ee6